### DISCUSSION TITLE
My colors  are off and not sure why?

### DISCUSSION THREAD

**Discussion by abayomi185** [olimorris/onedarkpro.nvim#153] [Q&A] [answered]
Hey olimorris, my colours for the `onedark` theme is weird in my terminal after the update that discontinued the former theme names. The other themes (`onedark_vivid`, etc.) seem to be fine. Could I get some help troubleshooting it?

**Comment by abayomi185** [comment, marked as answer]
Thought I'd put some photos here:

The reds are brighter than they should be?

### `onedark`
![Screenshot 2023-01-14 at 20 47 13](https://user-images.githubusercontent.com/21103047/212496071-cce8737a-bb60-4777-979e-5fac52841d6c.png)
### `onedark_vivid`
![Screenshot 2023-01-14 at 20 47 56](https://user-images.githubusercontent.com/21103047/212496079-9ad6b0c9-7a9e-4e48-a679-19229da99ed2.png)

**Reply by olimorris** [reply]
Can you raise a bug report with the `minimal.lua` config? I can't quite fathom why this would be

**Reply by abayomi185** [reply]
It works fine with `minimal.lua` so I'll check to see. I happen to use LunarVim but tested using `minimal.lua` with neovim.

Weird that vivid works and the other doesn't though.

**Reply by olimorris** [reply]
Does the theme have priority over other plugins? Wondering if there is another `onedark` plugin that ships with LunarVim

**Reply by abayomi185** [reply]
I think there happens to be another `onedark` in my config. I didn’t want to spend too long troubleshooting so I copied the `onedark` colours and used them as custom colours for `onedark_vivid`. So far so good

**Reply by abayomi185** [reply]
Fixed the issue I was having. Writing more details in case anyone stumbles on this. I cleaned out my lunarvim folder in `~/local/share/lunarvim` and reinstalled (presumably got rid of old cache and plugins). Copied my config back over and set a priority in `lazy.nvim`. It works as expected now.

**Reply by olimorris** [reply]
Thanks for posting your solution and glad it's working now.
